README.md

```
@@ -1,8 +1,199 @@
+The recommended way to install them is to use the `dev.py setup` command, which will download and install the right
+versions to the `~/fulfillment-service/.local` directory:
+
+    $ cd ~/fulfillment-service/repository
+    $ ./dev.py setup
+
+## Development
+
+To build the binary locally use `dev.py build`. Then you can run the `bin/main` binary directly, for example, to start
+the gRPC server:
+
+    $ bin/main start server --db-url=postgres://service:service123@localhost:5432/servcie"
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)